packages/client/components/ActivityLibrary/ActivityCard.tsx

```diff
@@ -44,14 +51,44 @@ export interface ActivityCardProps {
   badge?: React.ReactNode
   children?: React.ReactNode
   type?: MeetingTypeEnum
+  templateRef?: ActivityCard_template$key
 }
 
 export const ActivityCard = (props: ActivityCardProps) => {
-  const {className, theme, title, children, type, badge} = props
+  const {className, theme, title, children, type, badge, templateRef} = props
   const category = type && MEETING_TYPE_TO_CATEGORY[type]
+  const [showTooltip, setShowTooltip] = useState(false)
+  const [hoverTimeout, setHoverTimeout] = useState<NodeJS.Timeout | null>(null)
+
+  const template = useFragment(
+    graphql`
+      fragment ActivityCard_template on MeetingTemplate {
+        ...ActivityLibraryCardDescription_template
+      }
+    `,
+    templateRef ?? null
+  )
+
+  const handleMouseEnter = () => {
+    const timeoutId = setTimeout(() => {
+      setShowTooltip(true)
+    }, 750)
```

Comment:
When I trying to read about many templates one by one, personally, I feel it could be faster.

Reply:
Happy to make this 500ms instead. Let's see how Georg feels about the speed